docs/core/v3.0.rst

```diff
@@ -0,0 +1,1763 @@
+.. This file is in restructured text format: http://docutils.sourceforge.net/rst.html
+.. _zarr-core-specification-v3.0:
+
+======================================
+ Zarr core specification (version 3.0)
+======================================
+-------------------------------
+ Editor's draft 25 May 2022
+-------------------------------
+
+Specification URI:
+    https://purl.org/zarr/spec/core/3.0
+    
+Editors:
+    * Alistair Miles (`@alimanfoo <https://github.com/alimanfoo>`_), Wellcome Sanger Institute
+    * Jonathan Striebel (`@jstriebel <https://github.com/jstriebel>`_), Scalable Minds
+
+Issue tracking:
+    `GitHub issues <https://github.com/zarr-developers/zarr-specs/labels/core-spec-v3.0>`_
+
+Suggest an edit for this spec:
+    `GitHub editor <https://github.com/zarr-developers/zarr-specs/blob/core-protocol-v3.0-dev/docs/core/v3.0.rst>`_
```

Comment:
```suggestion
```

I propose to remove this part of the front matter, as we want to solicit reviews and suggestions via the current PR, not via issues.

Reply:
No objections. Could even point to the ZEP process as the definition of how to handle suggestions.

Reply:
I updated the links in #168 to the project board instead of issues, I think this would be fine for now. Marking as resolved here.